### PR TITLE
Remove and replace OMB guidance on Link page

### LINKED ISSUE #2511: Update Link page to remove references to OMB M-17-06
### Purpose of this issue
This issue closes #2474 reporting the outdated reference to OMB M-17-06 on the [USWDS Link component page](https://designsystem.digital.gov/components/link/). Here, we are outlining work related to updating the Link page to remove these outdated policy references.

### Summary of team's discussion of this issue
@finekatie updated the team on the state of the Link page on 2/28/2024. She shared this [Mural board analyzing opportunities for updates on the Link page](https://app.mural.co/t/gsa6/m/gsa6/1709139000067/13d3ab9683ddb24d52ad44d6fa1830ab947783db?sender=u0621280f41cdbc6a64e46748).

The team discussed the following points on 2/28/2024, which should guide page updates:

- Remove references to M-17-06 because M-23-22 rescinds the previous memo. OK to mention that M-23-22 rescinds M-17-06 in copy.
- Some of the guidance based on M-17-06 is still solid and not addressed by M-23-22, so it is ok to keep those best practices in our content.
- Restructure/reorganize "musts" as "shoulds": Guidance should generally remain the same, but where we are currently talking about what you "must" do and it is not referenced in M-23-22, it should become what you "should do."
- We do NOT need to replace the M-17-06 accordion with a M-23-22 accordion
- Under "Additional Information" heading remove old policy and add new one

### Work to be done

- [x] Draft copy updates for Links page
- [x] Fed approval of copy updates
- [x] Create a pull request for copy updates
- [ ] Fed final review

### PR DESCRIPTION
**Pull request by finekatie**
# Summary

_Updated the Link component page with new OMB guidance (M-23-22), removed old OMB guidance (M-17-06) and cleaned up some copy._

## Related issue

Closes #2511 

## Preview link

Preview link: https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/kf-Link-page-omb-update/components/link/

## Problem statement

_Current Link component page has outdated OMB guidance (M-17-06). Leaving it this way could misinform users about how they need to use links in USWDS. Need to have current OMB information and links for M-23-22._

## Solution

_This removes outdated OMB guidance and replaces with current OMB guidance for use of links. It provides current information and also refers to Digital.gov guidance._

## Major changes

_Removing all reference to OMB M-17-06. Updating guidance and removing "must have" requirements. Making copy changes to improve readability._

## Testing and review

_Need comparative review between old and new version of page._

## Dependency updates

| Dependency | Old      | New      |
| ---------- | -------- | -------- |
| axe-core   | 4.8.3    | 4.8.4    |
| express    | 4.18.2   | 4.19.1   |
| postcss    | 8.4.32   | 8.4.38   |
| sass       | 1.69.5   | 1.72.0   |
| snyk       | 1.1266.0 | 1.1285.0 |

### CODE REVIEW

**Comment by finekatie** [comment]
Hi all. I'm having an issue after attempting to delete the file for usability-must.md. We don't need it anymore, but I think it triggered some kind of error when I deleted and recomitted the branch. I can't get a preview link anymore.
@mahoneycm or @mejiaj , are you able to assist?

**Review by mejiaj — COMMENTED** [review]
**Before/After**

Are the green shields above the circle checks intentional?

image


image

**Comment by mejiaj** [comment]
@finekatie fixed build issues. I've assigned back to you.

Question on the new icon style above green checks. Just confirming this is intentional. Otherwise LGTM.

File: _components/link/link.md [hunk 1]
```diff
@@ -12,9 +12,6 @@ guidance:
     path: guidance/overview.md
     first: true
     html: true
-  - heading: "What you must do"
-    path: guidance/usability-must.md
-    first: true
```
[the remark this answers is not in the document]
Comment (finekatie) [reply _components/link/link.md:17]: Sometime you'll have to explain to me what that means. lol Glad you found it

**Comment by finekatie** [comment]
> @finekatie fixed build issues. I've assigned back to you.
> 
> Question on the new icon style above green checks. Just confirming this is intentional. Otherwise LGTM.

I fixed the icons and remerged. I think this is ready for fed final review. @mejiaj would you agree?

**Comment by mejiaj** [comment]
@finekatie yes, looks good. Icons are consistent now.

Build issues look fine too, the usual pa11y~ 😞

File: _components/link/link.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mejiaj) [reply _components/link/link.md:17]: We were pointing to something that no longer existed, so the site didn't know what to do.

**Review by mejiaj — APPROVED** [review]

**Comment by finekatie** [comment]
Hi @thisisdano and @annepetersen , this is ready for FFR. Please review and comment as needed. Hoping you find it ready for approval.

**Comment by annepetersen** [comment]
Also need to add an entry to `Latest updates`, such as "**Replaced outdated references to M-17-06 with references to M-23-22.** M-23-22 replaces M-17-06. Also fixed some inconsistencies and edited a few words for clarity."

**Comment by thisisdano** [comment]
Let's use "supersedes" here instead for technical accuracy?

>**Replaced outdated references to M-17-06 with references to M-23-22.** M-23-22 supersedes M-17-06. Also fixed some inconsistencies and edited a few words for clarity.

File: _components/link/guidance/usability-should.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-- **Clearly identify external links.** The external link icon () is a good way to communicate that a link is external.
+- **Clearly identify external links.** The external link icon (<svg class="usa-icon text-middle" aria-hidden="true" focusable="false" role="img"><use xlink:href="{{ site.baseurl }}/assets/img/sprite.svg#launch"></use></svg>) is a good way to communicate that a link is external.
```
Comment (mejiaj) [inline _components/link/guidance/usability-should.md:1]: **Result**

image

**Comment by finekatie** [comment]
> Let's use "supersedes" here instead for technical accuracy?
> 
> > **Replaced outdated references to M-17-06 with references to M-23-22.** M-23-22 supersedes M-17-06. Also fixed some inconsistencies and edited a few words for clarity.

Hey @mejiaj , where does this get updated? I don't see an .md file for the link page that relates to this.

**Review by thisisdano — APPROVED** [review]

**Comment by finekatie** [comment]
Hey @mahoneycm or @mejiaj , it looks like Dan approved this but didn't merge it. What needs to be done? Can I do something?

**Pull request merged**